src/Core/Addon/Module/ModuleManagerBuilder.php

```diff
@@ -124,8 +127,17 @@ public function buildRepository()
 
     private function __construct()
     {
+        global $kernel;
+        if (is_null($kernel)) {
+            $config = Yaml::parse(_PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'app'.DIRECTORY_SEPARATOR.'config'.DIRECTORY_SEPARATOR.'config.yml');
+            $baseUrl = $config['csa_guzzle']['clients']['addons_api']['config']['base_url'];
```

Comment:
And what if we need another parameter one day ? It would be better to send the whole config to the construct.

We would get something like:
```php
$config = Yaml::parse(_PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'app'.DIRECTORY_SEPARATOR.'config'.DIRECTORY_SEPARATOR.'config.yml');
$config = $config['csa_guzzle']['clients']['addons_api']['config'];
```